.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,155 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        
+        **NOTE** A large number of issues reported against Podman are often found to already be fixed in more current versions of the project.  Before reporting an issue, please verify the version you are running with `podman version` and compare it to the latest release documented on the top of Podman's [README.md](../README.md).If they differ, please update your version of Podman to the latest possible and retry your command before creating an issue.
+        
+        Also, there is a running list of known issues in the [Podman Troubleshooting Guide](https://github.com/containers/podman/blob/main/troubleshooting.md), please reference that page before opening a new issue.
+        
+        If you are filing a bug against `podman build`, please instead file a bug against Buildah (https://github.com/containers/buildah/issues). Podman build executes Buildah to perform container builds, and as such the Buildah maintainers are best equipped to handle these bugs.
+
+        Commands you might need to run to create the issue
+          $ podman version
+          $ podman info
+          $ rpm -q podman
+  - type: dropdown
+    id: podman_version
+    attributes:
+      label: Podman Version
+      description: What version of our podman are you running?
+      options:
+        - main
+        - 4.3
+        - 4.2
+        - 4.1
+        - 4.0
+        - 3.4
+        - 3.3
+        - 3.2
+        - 3.1
+        - 3.0
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: full_podman_version
+    attributes:
+      label: Full Podman Version
+      description: Please enter the full version
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System 
+      description: What operating system are you using?
```

Comment:
This, too, is in `podman info`. I don't see how this is a useful addition to our template.